routing.rst

```diff
@@ -531,6 +531,11 @@ and method: ``AppBundle\Controller\BlogController::showAction``. But if you
 follow some simple conventions, the logical name is more concise and allows
 more flexibility.
 
+.. tip::
+
+   For referring to the ``__invoke`` method of a controller, no need for
+   passing the method name, just use its fully-qualified namespace.
```

Comment:
👍  For the reword and the example, I made the changes.
About the visibility, IMHO we should stay consistent with the `controllers as a service` _note_ above, don't you think it's enough as is?